pkgs/by-name/de/debian-archive-keyring/package.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, gnupg
+, jetring
+, debian-archive-keyring
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "debian-archive-keyring";
+  version = "2023.4";
+
+  src = fetchFromGitLab {
+    domain = "salsa.debian.org";
+    owner = "release-team";
+    repo = "${finalAttrs.pname}";
+    rev = "${finalAttrs.version}";
+    hash = "sha256-RjHC4AJ0xyVmELVjX/WBmZYsKWFqcg8F5j+9+18jwj4=";
+  };
+
+  nativeBuildInputs = [
+    gnupg
+    jetring
+  ];
+
+  makeFlags = [
+    "DESTDIR=${placeholder "out"}"
```

Comment:
DESTDIR is not recommended.

Have you tried PREFIX instead?

Reply:
This one does not seem to support `PREFIX`: https://salsa.debian.org/release-team/debian-archive-keyring/-/blob/2023.4/Makefile#L66-71

Reply:
This is way worse. Nixpkgs does not install under `$out/usr`.
This is a case for patching or manual copy.